dali/base/dafdesc.cpp

```diff
@@ -1340,17 +1352,31 @@ class CFileDescriptor:  public CFileDescriptorBase, implements ISuperFileDescrip
             if (_interleaved)
                 *_interleaved = interleaved;
         }
-    }
 
+        const char * historyStr = "History";
```

Comment:
I think I'd use 2 bytes I think (unsigned short)
Define for current version static const unsigned short serializationVersion = 1;
and add a comment above the 2 existing #define's to say they are legacy and cannot be changed.